site/public/css/rotatingsectionsform.css

```diff
@@ -7,6 +7,17 @@ fieldset > * {
     margin-bottom: 0.5em;
 }
 
+#fewest-exclude {
+    margin-left: 13px;
+    border-left: solid #cccccc 1px;
```

Comment:
Instead of a specific color value here I think you'd want to use the CSS `var` functionality and use one of the already defined colors in `site/public/css/colors.css `